### PR TITLE
fix: Blocklist table optimised for all viewports

### PR DESCRIPTION
**Pull request by harshit078**
## Description

- This PR fixes the issue #7549 
- Optimised blocktable for all viewports

## Changes - 

Screenshot 2024-10-12 at 5 11 11 PM

https://github.com/user-attachments/assets/d5fa063d-2819-4a9d-a9b2-e3ceefe65c8d

### CODE REVIEW

File: packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsBlocklistTable.tsx [hunk 1]
```diff
@@ -16,21 +16,38 @@ const StyledTable = styled(Table)`
...
-const StyledTableBody = styled(TableBody)`
-  border-bottom: 1px solid ${({ theme }) => theme.border.color.light};
+const StyledTableHeader = styled(TableHeader)`
+  width: 100%;
+  @media (max-width: ${MOBILE_VIEWPORT}px) {
+    white-space: pre;
+    margin-left: ${({ theme }) => theme.spacing(6)};
+  }
+
+  @media (min-width: 360px) and (max-width: 390px) {
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsBlocklistTable.tsx:26]: have these 360 and 390px limit been discussed with design
As far as I know we only have the MOBILE_VIEWPORT limit on the project right now.

I'm not against introducing behaviors for 360px but we need to have a project-wide approach in this case.
Can we revert this change and get the input of @Bonapara on the overall approach?

Comment (harshit078) [reply packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsBlocklistTable.tsx:26]: @charlesBochet sure, I'll revert the min width code blocks

**Review by charlesBochet — APPROVED** [review]
@harshit078 I have found a simpler way to achieve a nice result
